mautic/mautic
# Add error message to new/update contact

**Please be sure you are submitting this against the _staging_ branch.**

[//]: # ( Please answer the following questions: )

| Q  | A
| --- | ---
| Bug fix? | 
| New feature? | 
| Automated tests included? |
| Related user documentation PR URL | 
| Related developer documentation PR URL | 
| Issues addressed (#s or URLs) | 
| BC breaks? | 
| Deprecations? | 

[//]: # ( Note that all new features should have a related user and/or developer documentation PR in their respective repositories. )

[//]: # ( Required: )
#### Description:
We can't save contact profile, because some of the fields was mandatory. And didn't see context in form, looks like profile was saved correctly. 
This PR added error message, If some fields return error.

[//]: # ( As applicable: )
#### Steps to reproduce the bug:
1. Create custom field and set it as required
2. Edit contact, save and see form, looks like profile was update, but not

#### Steps to test this PR:
1. Load up [this PR](https://mautibox.com)
2. Repeat all steps
3. See error message in right corner

![image](https://user-images.githubusercontent.com/462477/60096319-ec59d980-9750-11e9-8ead-0ec1b7ca9deb.png)
